lib/mosql/streamer.rb

```diff
@@ -170,14 +174,31 @@ def optail
       if tail_from.is_a? Time
         tail_from = tailer.most_recent_position(tail_from)
       end
+
+      last_batch_insert = Time.now
       tailer.tail(:from => tail_from)
       until @done
         tailer.stream(1000) do |op|
```

Comment:
Since the author is inactive for long time, I create a branch based on this where I added the @nelhage suggestions: https://github.com/stripe/mosql/pull/137
I hope someone can see this.